climada/hazard/test/test_flood.py

```diff
@@ -78,18 +78,6 @@ def test_exact_area_selection_country(self):
         self.assertAlmostEqual(testCentroids.lat[11], 47.2289138)
         self.assertAlmostEqual(testCentroids.lat[12], 47.2289138)
 
-    def test_exact_area_selection_region(self):
-
-        testCentr, isos, natIDs = RiverFlood._select_exact_area(reg=['SWA'])
-
-        self.assertEqual(testCentr.shape, (877, 976))
-        self.assertAlmostEqual(np.min(testCentr.lat), -0.68767620000001, 4)
-        self.assertAlmostEqual(np.max(testCentr.lat), 38.43726119999998, 4)
-        self.assertAlmostEqual(np.min(testCentr.lon), 60.52061519999998, 4)
-        self.assertAlmostEqual(np.max(testCentr.lon), 101.1455501999999, 4)
-        self.assertAlmostEqual(testCentr.lon[10000], 98.27055479999999, 4)
-        self.assertAlmostEqual(testCentr.lat[10000], 11.47897099999998, 4)
-
```

Comment:
Why did you decide to remove this? Isn't this a valid test case?

Reply:
It is a valid test case, but this function refers to the selection of an entire region, so it is no longer possible with a test file covering only two countries.

Reply:
I further don't think that the functionality of choosing an entire region instead of single countries is so important that it is worth having a larger test-file in there, that's why I decided to simply remove it instead of cutting an entire region as a test file.

Reply:
Maybe there's a misunderstanding what this function tests, because I think it doesn't rely on the test-file at all. It only loads grid points (not the values, only the coordinates) and checks whether the right grid has been loaded and truncated.

Reply:
Yes, you are right. I will bring it back in.

Reply:
Yes, you are of course right. Thanks, for pointing this out.